scripts/goreleaser-post-actions.sh

```diff
@@ -17,6 +17,14 @@ for f in dist/*.{tar.gz,deb,rpm,txt}; do
 done
 
 # Create the actual Github Release.
+if [[ -z $(git config --get user.email) ]]; then
```

Comment:
Why are we adding `git config` to this script?

Reply:
I've pushed another commit: 614d377 and 4dc17a8, which only would set this if the environment variable `CI` is set to `true`. This will only happen within the GitHub action which is executing the release.

If you check the execution of the release here:
https://github.com/elastic/ecctl/runs/665265418?check_suite_focus=true, you'll see that the `git commit` fails because the git configuration isn't set.

The actual script which. runs that is in [`scripts/update-brew-tap.sh`](https://github.com/elastic/ecctl/blob/master/scripts/update-brew-tap.sh)

Reply:
Rather than changing the configuration, you can use environment variables to accomplish the same thing. 

https://git-scm.com/book/en/v2/Git-Internals-Environment-Variables

We do this with the cloud cut-release jobs: 
https://github.com/elastic/cloud/blob/e71b84ab776c2638d63da83e1c3e96537a8d6902/.ci/jobs/cut-release-ece.yml#L151-L153

In out case however, [we leverage a Jenkins plugin](https://github.com/elastic/cloud/blob/e71b84ab776c2638d63da83e1c3e96537a8d6902/.ci/jobs/cut-release-ece.yml#L137) to set the values to the person who executed the cut release job so that we can have a more rich commit history.

Either way you do it -- you shouldn't need to modify the git configuration with the environment variables.

Reply:
ah that's cool. I'll correct it then!

Reply:
I addressed your comments in c580e75

Reply:
Actually, I added them to the `env` object in the `yaml` specification for the release action (b37494c).